### PR TITLE
code clean up of robot_model

### PR DESCRIPTION
**Pull request by bailaC**
### Description

The repetitive namespace declaration of "moveit::core" has been removed from all the spaces. 

### Checklist
- [x] **Required**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extended the tutorials / documentation, if necessary [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Include a screenshot if changing a GUI
- [ ] Optional: Created tests, which fail without this PR [reference](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/tests.html)
- [ ] Optional: Decide if this should be cherry-picked to other current ROS branches (Indigo, Jade, Kinetic)

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_core/robot_model/src/fixed_joint_model.cpp [hunk 1]
```diff
@@ -36,63 +36,67 @@
 
 #include <moveit/robot_model/fixed_joint_model.h>
 
-moveit::core::FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
+namespace moveit
+{
+namespace core
+{
+FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
```
Comment (davetcoleman) [inline moveit_core/robot_model/src/fixed_joint_model.cpp]: add space after curly bracket and before function

Comment (davetcoleman) [reply moveit_core/robot_model/src/fixed_joint_model.cpp]: ...unless clang-format does not allow this. do this for all the files in this PR please.

File: moveit_core/robot_model/src/robot_model.cpp [hunk 2]
```diff
@@ -47,17 +47,19 @@
 #include "order_robot_model_items.inc"
 
 /* ------------------------ RobotModel ------------------------ */
```
Comment (davetcoleman) [inline moveit_core/robot_model/src/robot_model.cpp]: this comment above is odd, can you remove that too please?

File: moveit_core/robot_model/src/fixed_joint_model.cpp (hunk 1 above)
Comment (bailaC) [reply moveit_core/robot_model/src/fixed_joint_model.cpp]: Are you saying to do the change as following?
```
namespace moveit
{
namespace core
{

FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
```

Comment (bailaC) [reply moveit_core/robot_model/src/fixed_joint_model.cpp]: Well, clang-format is not allowing to do the change i mentioned earlier.

File: moveit_core/robot_model/src/robot_model.cpp (hunk 2 above)
Comment (bailaC) [reply moveit_core/robot_model/src/robot_model.cpp]: deleted the comment.

**Comment by bailaC** [comment]
Yes @rhaschke . I did "Replace all" from my code editor and also did visual checking for all files. Thanks for approving.

**Pull request merged**